.travis.yml

```diff
@@ -0,0 +1,15 @@
+sudo: required
+
+services:
+  - docker
+
+addons:
+  apt:
+    packages:
+      - make
+
+script:
+- docker build --tag bitbox-wallet-dev .
+- mv Dockerfile Dockerfile.dev
```

Comment:
i think you can specify the dockerfile with `docker build -f Dockerfile.travis`, so you don't have to do this awkward file swap :)

Reply:
Agreed.